reana_server/config.py

```diff
@@ -116,6 +116,11 @@ def _(x):
 APP_DEFAULT_SECURE_HEADERS["content_security_policy"] = {}
 APP_HEALTH_BLUEPRINT_ENABLED = False
 
+# Rate limiting configuration
+# ===========================
+RATELIMIT_AUTHENTICATED_USER = "20 per second"
```

Comment:
Please amend `CHANGES.rst` and say something like:

- Changes rate limiting defaults to allow up to 20 connections per second.